client/lib/source/poll-source.ts

```diff
@@ -15,7 +16,7 @@ export class PollSource<T> implements Source<T> {
     private readonly http: Http<T>,
   ) { }
 
-  watch$ (store: Store<T>): Observable<Update<T>> {
+  watch$ (store: Store<T>): Observable<RPCResponse<Update<T>>> {
```

Comment:
We changed this back because we go from throwing an error (which will stop the stream) to the enum error type returned from the rust side (Still get errors without killing)